presto-native-execution/presto_cpp/main/types/PrestoToVeloxExpr.h

```diff
@@ -14,10 +14,19 @@
 #pragma once
 
 #include <stdexcept>
+#include "presto_cpp/main/common/Configs.h"
 #include "presto_cpp/main/types/TypeParser.h"
 #include "presto_cpp/presto_protocol/core/presto_protocol_core.h"
 #include "velox/core/Expressions.h"
 
+namespace {
+inline std::string formatDefaultNamespacePrefix(
```

Comment:
Its bit odd this function has name formatDefaultNamespacePrefix and then takes the default namespace prefix as a parameter. Maybe you should call this addDefaultNamespace and pick up the default namespace in the function without passing it as a parameter everywhere.

Reply:
Yes, calling this `addDefaultNamespace` makes sense. 
I am confused about the picking up of the default namespace from the functions part, the functions passed down to the function won't have any namespace attached.

For eg: 
`presto.default.$operator.$add` should map to` {namespace-prefix}.add` , in this case `native.default.add`
https://github.com/prestodb/presto/blob/40161c0a0b68ee0842ad25a66d849d6425e2e883/presto-native-execution/presto_cpp/main/types/PrestoToVeloxExpr.cpp#L41 

The call to the formatDefaultNamespacePrefix() function will take in the default namespace prefix and the function name, add the default namespace prefix to the function name and return the string.